apps/i18n/javalab/en_us.json

```diff
@@ -33,7 +33,7 @@
   "compilationSuccess": "Compilation successful.",
   "compiled": "Compiled!",
   "compileFailed": "Compile Failed.",
-  "compilerError": "We couldn't compile your program. Look for bugs in your program and try again.",
+  "compilerError": "We couldn't compile your program. Look for bugs in your program and try again. You can also check if the AI Tutor has any helpful hints.",
```

Comment:
"You can ask AI Tutor" bit needs to only show if the AI Tutor is enabled for the student/level